app/containers/RoomTypeIcon/index.tsx

```diff
@@ -24,44 +25,38 @@ interface IRoomTypeIcon {
 	sourceType?: IOmnichannelSource;
 }
 
-const RoomTypeIcon = React.memo(({ type, isGroupChat, status, style, teamMain, size = 16, sourceType }: IRoomTypeIcon) => {
-	const { theme } = useTheme();
+const RoomTypeIcon = React.memo(
+	({ userId, type, isGroupChat, status, style, teamMain, size = 16, sourceType }: IRoomTypeIcon) => {
+		const { theme } = useTheme();
 
-	if (!type) {
-		return null;
-	}
+		if (!type) {
+			return null;
+		}
 
-	const color = themes[theme].titleText;
-	const iconStyle = [styles.icon, { color }, style];
+		const iconStyle = [styles.icon, style];
 
-	if (type === 'd' && !isGroupChat) {
-		if (!status) {
-			status = 'offline';
+		if (type === 'd' && !isGroupChat && userId) {
+			return <Status id={userId} style={iconStyle} size={size} status={status} />;
```

Comment:
Do you know if we still need to pass this status? If this prop `status` is been passed only for the omnichannel's room, isn't it?

Reply:
It's going to be required on MessageComposer